helm/nessie/templates/configmap.yaml

```diff
@@ -0,0 +1,126 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "nessie.fullname" . }}
+  labels:
+    {{- include "nessie.labels" . | nindent 4 }}
+    {{- if .Values.configMapLabels }}
+    {{- toYaml .Values.configMapLabels | nindent 4 }}
+    {{- end }}
+data:
+  application.properties: |-
+    {{- $map := dict -}}
+    {{- $_ := set $map "nessie.version.store.type" .Values.versionStoreType -}}
+
+    {{- if .Values.logLevel -}}
+    {{- $_ = set $map "quarkus.log.level" .Values.logLevel -}}
+    {{- $_ = set $map "quarkus.log.console.level" .Values.logLevel -}}
+    {{- $_ = set $map "quarkus.log.file.level" .Values.logLevel -}}
+    {{- if or (eq .Values.logLevel "DEBUG") (eq .Values.logLevel "TRACE") -}}
+    {{- $_ = set $map "quarkus.log.min-level" .Values.logLevel -}}
+    {{- end -}}
+    {{- end -}}
+
+    {{- if or (eq .Values.versionStoreType "DYNAMODB") (eq .Values.versionStoreType "DYNAMO") -}}
+    {{- if .Values.dynamodb.region -}}
+    {{- $_ = set $map "quarkus.dynamodb.aws.region" .Values.dynamodb.region -}}
+    {{- end -}}
+    {{- if .Values.dynamodb.profile -}}
+    {{- $_ = set $map "quarkus.dynamodb.aws.credentials.profile-provider.profile-name" .Values.dynamodb.profile -}}
+    {{- end -}}
+    {{- end -}}
+
+    {{- if eq .Values.versionStoreType "CASSANDRA" -}}
+    {{- $_ = set $map "quarkus.cassandra.keyspace" .Values.cassandra.keyspace -}}
+    {{- $_ = set $map "quarkus.cassandra.contact-points" .Values.cassandra.contactPoints -}}
+    {{- if .Values.cassandra.localDatacenter -}}
+    {{- $_ = set $map "quarkus.cassandra.local-datacenter" .Values.cassandra.localDatacenter -}}
+    {{- end -}}
+    {{- if .Values.cassandra.auth.username -}}
+    {{- $_ = set $map "quarkus.cassandra.auth.username" .Values.cassandra.auth.username -}}
+    {{- $_ = set $map "quarkus.cassandra.auth.password" .Values.cassandra.auth.password -}}
```

Comment:
(side note) Should we move this to a secret?

Reply:
Absolutely. I took note of that already.